### ISSUE TITLE
Questions regarding the required imgs

### ISSUE THREAD

**Issue by Lucas-C** [danielholmes/bga-workbench#2]
Hi

The `bgawb validate` command requires that some images are present in the repo:
https://github.com/danielholmes/bga-workbench/blob/master/src/BGAWorkbench/Project/Project.php#L222

From the documentation, the `game_box75.png` & `game_box180.png` images are auto-generated:
http://en.doc.boardgamearena.com/Game_art:_img_directory#Requested_images

Shouldn't they be optionnal ?

Also, there seem to be a name mismatch: `game_box75.png` in your project vs `game_box72.png` on the BGA wiki. Is there a reason ?

Thanks

**Comment by danielholmes** [comment]
> Shouldn't they be optionnal ?

I think you're right. I'll make this change

> Also, there seem to be a name mismatch: game_box75.png in your project vs game_box72.png on the BGA wiki. Is there a reason ?

I'm pretty sure this is a typo in the wiki. I've looked at several example projects (hearts, gomoku, etc) and another production project - they all have game_box75. Also battle is working with game_gox75

**Comment by Lucas-C** [comment]
Thank you very much !

You can close this ticket as you like.

**Comment by Lucas-C** [comment]
Just FYI, with an `img/game_box75.png` file only, `bgawb validate` generates the following error:
```
The command "cp img/game_box72.png img/game_box75.png" exited with 1.
```

**Comment by danielholmes** [comment]
Have released a new version solving this (#7219824)
